docs/contributing.rst

```diff
@@ -2,12 +2,122 @@
 
 .. _contributing:
 
+.. toctree::
+ :maxdepth: 2
+ :numbered:
+
 
 Contributor's Guide
 ===================
 
-Anyone can contribute to and participate in the Uxarray project
-at any levels of project development! We conduct all of our work
-in the open, and all of our work is Open Source Licensed.
+Welcome to the Contributor's Guide for UXarray!
+
+So, welcome to the team! If you are reading this document, we hope that
+you are already or soon-to-be a UXarray contributor! Please keep reading!
+
+1. Overview
+-----------
+
+UXarray is a community-owned, open-development effort that is a result
+of the collaboration between `Project Raijin <https://raijin.ucar.edu/>`_
+and DOE’s `SEATS Project <https://seatstandards.org/>`_ ]. Even though the
+Uxarray team has initiated and been expanding this package, outside
+contributions are welcome and vital for its long-term sustainability.
+Therefore, we invite other community members to become part of this
+collaboration at any level of contribution.
+
+1.1. Why the name "UXarray"?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We have created UXarray by composing `Xarray <https://docs.xarray.dev/en/stable/>`_,
+a Pangeo ecosystem package commonly-used  for structured grids
+recognition, to support reading and recognizing unstructured grid
+model outputs. We picked the name "UXarray" and preferred to
+capitalize the first two letters to emphasize it is Xarray for
+Unstructured grids.
```

Comment:
```suggestion
We have created UXarray by composing `Xarray <https://docs.xarray.dev/en/stable/>`_,
a Pangeo ecosystem package commonly-used for structured grids
recognition, to support reading and recognizing unstructured grid
model outputs. We picked the name "UXarray" and preferred to
capitalize the first two letters to emphasize it is Xarray for
Unstructured grids.
```

I'm not sure what you mean by "composing" here

Reply:
I (was trying to) refer to the OOP concept, composition, for Xarray dataset, but the word "composing" apparantly won't work for that. How about something like this:

"We have created UXarray based on `Xarray <https://docs.xarray.dev/en/stable/>`_ (via composition of a Xarray dataset object) ..."

Reply:
Oh! Yes that is much more clear for me